admin-perspective/olm/install-forcenamespace/install-forcenamespace.md

```diff
@@ -0,0 +1,90 @@
+
+This design is now surfacing both the watch (availability) namespace and the install namespaces to the user. The installed namespace is initially read-only and defaults to **openshift-operators** when the availability namespace is **all namespaces**, and defaults to the **value from watch namespace dropdown** when the availability namespace is **single** & has a namespace selected. The user has the option to **Edit** the installation namespace if desired (and the operator allows.)
+
+This design is not intended to convey every combination of controls that is possible via operator InstallModes, that information is available via [this spreadsheet](https://docs.google.com/spreadsheets/d/19J7IEjORg4-U6nQEwQkxPwe3SJazgpobN2HFKE2rFcc/edit?folder=1y5NhdQNm8b-NeX1I0uN6PHQzhp_NJPh0#gid=0). Controls in this design will be enabled/disabled/hidden as described in the spreadsheet and as conveyed in the screens below and the existing UI’s paradigms. This design is intended to show the location of controls and the interactions between them.
+
+This design also captures the ability to enable cluster monitoring on namespaces that are installed to and the operator requests it. This setting applies to only certain whitelisted namespaces, and the user should be warned of the implications of doing so. Operators can request monitoring be enabled on their install namespace but we will also present this option.
+
+## Installing with no operator-defined namespace (existing behavior)
+
+![Operator namespace not defined all](img/1-1-noDefinedNS-all.png)
+- Installing an operator without a operator-defined namespace now conveys that by default the operator would be installed on the **openshift-operators** namespace with the option to **Edit**.
+- Namespaces available to install to would not yet have an OperatorGroup object present configured to watch all namespaces.
+
+![Operator namespace not defined single](img/1-2-noDefinedNS-single.png)
+-A specific namespace may be chosen for the operator to be available to (and installed to.)
+
+![Operator namespace not defined single](img/1-3-noDefinedNS-single.png)
+-The install namespace will match the specific namespace the operator is available to, until the install namespace is manually changed.
+
+![Operator namespace not defined single](img/1-4-noDefinedNS-single.png)
+- The user can **Edit** to modify the namespace the operator is installed to.
+
+![Operator namespace not defined single](img/1-5-noDefinedNS-single.png)
+- The user manually sets an alternative namespace to install to.
+
+![Operator namespace not defined single](img/1-6-noDefinedNS-single.png)
+- Changing the availability namespace no longer has any affect on the install namespace, as it has been manually changed.
+
+## Installing with an operator-defined namespace
```

Comment:
This may sound like an annoying last-minute idea, but could we do a `?` that helps explain the differences between "available in" and "installed in". I know it is really clear to us, but I don't' think it will be for new users.

Reply:
Included the discussed hint text for these headers in the description of the first image in design. This text will be surfaced using the existing popover style.
![image](https://user-images.githubusercontent.com/54860125/72748821-47886900-3b86-11ea-96f3-ea98d6e3257f.png)